guides/common/modules/proc_installing-the-scc-manager.adoc

```diff
@@ -1,14 +1,33 @@
 [id="Installing_the_SCC_Manager_Plugin_{context}"]
 = Installing the SCC Manager Plug-in
 
-Perform the following step to install the _SCC Manager_ plug-in on your {Project}.
+Perform the following steps to install the _SCC Manager_ plug-in on your {Project}.
 
 .Procedure
-. Install the SCC Manager plug-in on your {ProjectServer}:
+. Connect to your {ProjectServer} using SSH:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-scc-manager
+# ssh root@{foreman-example-com}
+----
+. Install the SCC Manager plug-in on your {Project}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {package-install-project} -y rubygem-foreman_scc_manager
```

Comment:
Seems the rebase on https://github.com/theforeman/foreman-documentation/pull/1800 went wrong ;)

Reply:
I rebased it and went with Ewoud's change. I hope, it is resolved. :crossed_fingers:

Reply:
Doesn't look like it. You reverted all the changes.

Reply:
Oww gosh, I accepted the incoming changes. :face_with_spiral_eyes:

Reply:
Fixed :+1: